### PR TITLE
enhancement: add name info for filters

### PR DESCRIPTION
**Pull request by rambohe-ch**
/kind enhancement

#### What this PR does / why we need it:
Add name info for filters in yurthub, meanwhile, add filter name in the log message in order to monitor status of the filter.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/yurthub/filter/interfaces.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 )
 
 type Interface interface {
+	Name() string
```
Comment (Fei-Guo) [inline pkg/yurthub/filter/interfaces.go:29]: Is the change in the interface necessary?

Comment (rambohe-ch) [reply pkg/yurthub/filter/interfaces.go:29]: @Fei-Guo ok, i have removed the new func `Name()`.

**Pull request merged**